### PR TITLE
documentation for license plate detector

### PR DESCRIPTION
**Pull request by ongzizhao**
Documentation for license plate detector

### CODE REVIEW

File: docs/source/use_cases/privacy_license_plate.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+There are two main components to license plate anonymisation: 1) license plate detection using AI; and 2) license plate de-identification.
+
+**1. License Plate Detection**
+
+We use an open source object detection model known as [YoloV4](https://arxiv.org/abs/2004.10934) to identify license plate. This allows the application to identify the location of license plate in an image/video feed. The location of detected license plates are returned as an array of coordinates in the form [x1, y1, x2, y2], where (x1, y1) is the top-left corner of the bounding box, and (x2, y2) is the bottom-right. These are used to form the bounding box of each license plate detected. For more information on how adjust the license plate detector node, checkout the [license plate detector configurable parameters](/peekingduck.pipeline.nodes.model.licenseplate.Node).
+
+**2. License Plate De-Identification**
+
+To perform license plate de-identification, we use gaussian blur to blur the pixels in the the area bounded by the bounding boxes.
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:27]:
``` suggestion
To perform license plate de-identification, we blur the pixels in the area bounded by the bounding boxes using a Gaussian function (Gaussian blur).
```

File: docs/source/use_cases/privacy_license_plate.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    model_type: v4tiny
+- dabble.fps
+- draw.blur_bbox
+- draw.legend
+- output.screen
+```
+
+**1. License Plate Detection Node**
+
+As mentioned, we use a custom trained Yolov4 model for license plate detection. By default, it uses the Yolov4-tiny model to detect the license plate. For better accuracy, you can change the parameters in the run config declaration to use the Yolov4 model instead.
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:46]: "custom-trained"

File: docs/source/use_cases/privacy_license_plate.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+> peekingduck run --config_path <path_to_privacy_license_plate.yml>
+```
+
+## How it Works
+
+There are two main components to license plate anonymisation: 1) license plate detection using AI; and 2) license plate de-identification.
+
+**1. License Plate Detection**
+
+We use an open source object detection model known as [YoloV4](https://arxiv.org/abs/2004.10934) to identify license plate. This allows the application to identify the location of license plate in an image/video feed. The location of detected license plates are returned as an array of coordinates in the form [x1, y1, x2, y2], where (x1, y1) is the top-left corner of the bounding box, and (x2, y2) is the bottom-right. These are used to form the bounding box of each license plate detected. For more information on how adjust the license plate detector node, checkout the [license plate detector configurable parameters](/peekingduck.pipeline.nodes.model.licenseplate.Node).
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:23]:
```suggestion
We use an open-source object detection model known as [YoloV4](https://arxiv.org/abs/2004.10934) to identify license plates. This allows the application to identify the location of the license plate in an image/video feed. The locations of detected license plates are returned as an array of coordinates in the form [x1, y1, x2, y2], where (x1, y1) is the top-left corner of the bounding box, and (x2, y2) is the bottom-right. These are used to form the bounding box of each license plate detected. For more information on how to adjust the license plate detector node, check out the [license plate detector configurable parameters](/peekingduck.pipeline.nodes.model.licenseplate.Node).

```

File: docs/source/use_cases/privacy_license_plate.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+To try our solution on your own computer with [PeekingDuck installed](../getting_started/01_installation.md): use the following configuration file: [privacy_license_plate.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/privacy_license_plate.yml) and run PeekingDuck.
+
+```
+> peekingduck run --config_path <path_to_privacy_license_plate.yml>
+```
+
+## How it Works
+
+There are two main components to license plate anonymisation: 1) license plate detection using AI; and 2) license plate de-identification.
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:19]:
```suggestion
There are two main components to license plate anonymisation: 1) license plate detection using AI, and 2) license plate de-identification.
```

File: docs/source/use_cases/privacy_license_plate.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## Overview
+
+As organisations collect more data, there is a need to better protect the identities of individuals in public and private places. AI Singapore has developed a solution that performs face anonymisation. This can be used to comply with the General Data Protection Regulation (GDPR) or other data privacy laws.
+
+Our solution automatically detects and blurs license plate of cars. This is further elaborated in a [subsequent section](#how-it-works).
+
+## Demo
+
+To try our solution on your own computer with [PeekingDuck installed](../getting_started/01_installation.md): use the following configuration file: [privacy_license_plate.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/privacy_license_plate.yml) and run PeekingDuck.
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:11]:
```suggestion
To try our solution on your computer with [PeekingDuck installed](../getting_started/01_installation.md): use the following configuration file: [privacy_license_plate.yml](https://github.com/aimakerspace/PeekingDuck/blob/dev/use_cases/privacy_license_plate.yml) and run PeekingDuck.
```

File: docs/source/use_cases/privacy_license_plate.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+- output.screen
+```
+
+**1. License Plate Detection Node**
+
+As mentioned, we use a custom trained Yolov4 model for license plate detection. By default, it uses the Yolov4-tiny model to detect the license plate. For better accuracy, you can change the parameters in the run config declaration to use the Yolov4 model instead.
+
+**2. License Plate De-Identification Nodes**
+
+You can mosaic or blur the faces detected using the `draw.mosaic_bbox` or `draw.blur_bbox` in the run config declaration.
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:50]:
```suggestion
You can choose to mosaic or blur the detected license plates using the `draw.mosaic_bbox` or `draw.blur_bbox` node in the run config declaration.
```

File: docs/source/use_cases/privacy_license_plate.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+**3. Adjusting Nodes**
+
+With regard to the Yolov4 model, some common node behaviours that you might want to adjust are:
```
Comment (chngzyk) [inline docs/source/use_cases/privacy_license_plate.md:54]:
```suggestion
With regard to the Yolov4 model, some common node configurations that you might want to adjust are:
```

File: docs/source/use_cases/privacy_license_plate.md (hunk 1 above)
Comment (ongzizhao) [reply docs/source/use_cases/privacy_license_plate.md:27]: changes made accordingly

File: docs/source/use_cases/privacy_license_plate.md (hunk 5 above)
Comment (ongzizhao) [reply docs/source/use_cases/privacy_license_plate.md:11]: changes made accordingly

File: docs/source/use_cases/privacy_license_plate.md (hunk 6 above)
Comment (ongzizhao) [reply docs/source/use_cases/privacy_license_plate.md:50]: changes made accordingly

File: docs/source/use_cases/privacy_license_plate.md (hunk 7 above)
Comment (ongzizhao) [reply docs/source/use_cases/privacy_license_plate.md:54]: changes made accordingly

**Pull request merged**